front/app/modules/commercial/custom_style/index.tsx

```diff
@@ -1,10 +1,7 @@
 import { ModuleConfiguration } from 'utils/moduleUtils';
-import CustomStyleInputs from './admin/components/CustomStyleInputs';
 
 const configuration: ModuleConfiguration = {
-  outlets: {
-    'app.containers.Admin.settings.customize.fields': CustomStyleInputs,
-  },
+  outlets: {},
```

Comment:
Module still to be removed in front/back, moved the overlay settings to the core.